bindings/Python/CMakeLists.txt

```diff
@@ -13,16 +13,29 @@ if(ADIOS2_HAVE_MPI)
   target_link_libraries(adios2py PRIVATE PythonModule::mpi4py)
 endif()
 
-set(CMAKE_INSTALL_PYTHONDIR
-  "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages"
-)
+if(WIN32)
+  set(CMAKE_INSTALL_PYTHONDIR
+    "${CMAKE_INSTALL_LIBDIR}\\site-packages"
+  )
+else()
+  set(CMAKE_INSTALL_PYTHONDIR
+    "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages"
+  )
+endif()
 set(CMAKE_INSTALL_PYTHONDIR "${CMAKE_INSTALL_PYTHONDIR}"
   CACHE INTERNAL "" FORCE
 )
+set(CMAKE_PYTHON_OUTPUT_DIRECTORY
+  "${ADIOS2_BINARY_DIR}/${CMAKE_INSTALL_PYTHONDIR}"
+  CACHE PATH "Build directory for python modules"
+)
 set_target_properties(adios2py PROPERTIES
   OUTPUT_NAME adios2
-  LIBRARY_OUTPUT_DIRECTORY ${ADIOS2_BINARY_DIR}/${CMAKE_INSTALL_PYTHONDIR}
-  RUNTIME_OUTPUT_DIRECTORY ${ADIOS2_BINARY_DIR}/${CMAKE_INSTALL_PYTHONDIR}
+  ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_PYTHON_OUTPUT_DIRECTORY}
```

Comment:
funnily, this line is needed although one always builds a shared lib for python modules. well.

Reply:
We shouldn't actually need the ARCHIVE_OUTPUT_DIRECTORY.  Since this is built as a `MODULE` target and not `SHARED` then the only output is a `.dll` without a corresponding `.lib` export library.

Reply:
Yes, I know it's confusing but doesn't work without this also being set on multi-config generators due to a name clash at configure time,